packages/website/components/navigation/navigation.js

```diff
@@ -197,26 +197,60 @@ export default function Navigation({ isProductApp }) {
               </div>
 
               <div className={clsx('nav-items-wrapper', theme)}>
-                {navItems.map(item => (
-                  <Link passHref key={item.text} href={item.url}>
-                    <a
-                      href="replace"
-                      className={clsx('nav-item', item.url === router.route ? 'current-page' : '')}
-                      onClick={onLinkClick}
-                      onKeyPress={e => handleKeySelect(e, item.url)}
-                    >
-                      {item.text}
-                    </a>
-                  </Link>
+                {navItems.map((item, i) => (
+                  <>
+                    {item.links ? (
+                      <div className="nav-dd-button">
+                        <Link passHref key={item.text} href={item.url}>
```

Comment:
you should be able to use our new `Link` component so that you don't need to nest an anchor tag.  This goes for the other Links in here as well (it's possible your branch is a bit out of date, since the removed code now uses the new Link component).